src/collections/pages/en/index.md

```diff
@@ -1,6 +1,13 @@
 ---
 title: Inclusive Standards
 permalink: /en/
-lang: en
 layout: layouts/base
+hasBanner: true
+bannerBGColor: indigo-700
+bannerTitle: <b>Inclusive</b> Standards
+bannerBody: Inclusive Standards is a project led by the Inclusive Design Centre, dedicated to creating guidelines that ensure products, services, and environments are accessible and usable by everyone. We focus on promoting equity, diversity, and inclusion by developing standards that meet the needs of all individuals, including people with disabilities and marginalized communities.
+displayProjects: true
+displayEvents: true
+displayResources: true
+displayNews: true
```

Comment:
I think these should be removed and display should be determined by whether there is content.